core/commands/config.go

```diff
@@ -293,6 +294,92 @@ can't be undone.
 	},
 }
 
+var configProfileCmd = &cmds.Command{
+	Helptext: cmdkit.HelpText{
+		Tagline: "Apply profiles to config.",
+	},
+
+	Subcommands: map[string]*cmds.Command{
+		"apply":  configProfileApplyCmd,
+		"revert": configProfileRevertCmd,
```

Comment:
Actually reverting is impossible (for us). We're not reverting, we're applying the inverse of the profile. Do we really need to provide this feature?

We could, instead, provide inverse profiles: local-discovery and flatfs-ds.

I'm worried that a user will think that profiles act like drop-ins (that's how a lot of unix daemons work) and will believe  that revert will revert the apply.

Reply:
I agree with @Stebalien this is not really a revert so we perhaps we should name it something else, not sure what, "unapply" comes to mind.

The idea of instead providing inverse profiles is also a good idea.

Reply:
I think I like the idea of inverse profiles better, I'll see how that looks